### DISCUSSION TITLE
Does cpptools support using precompiled headers?

### DISCUSSION THREAD

**Discussion by celonymire** [microsoft/vscode-cpptools#8338] [Ideas]
CMake has added support for precompiled headers since 3.16 (https://cmake.org/cmake/help/latest/command/target_precompile_headers.html), could cpptools utilize such a feature to speed up compilation?

**Comment by sean-mcmanus** [comment]
I created a feature request at https://github.com/microsoft/vscode-cpptools/issues/8355 .

It's a feature that exists in VS that hasn't been ported to VS Code yet.
